src/magic_uv/op/copy_paste_uv.py

```diff
@@ -49,10 +49,7 @@ def _is_valid_context(context):
         return False
 
     # only 'VIEW_3D' space is allowed to execute
-    for space in context.area.spaces:
-        if space.type == 'VIEW_3D':
-            break
-    else:
+    if not common.is_valid_space(context, ['IMAGE_EDITOR', 'VIEW_3D']):
```

Comment:
I think this operation should be executed only in View3D.

```suggestion
    if not common.is_valid_space(context, ['VIEW_3D']):
```